bundles/org.openhab.binding.goecharger/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -21,15 +21,17 @@
 			<channel id="powerL1" typeId="pl1"/>
 			<channel id="powerL2" typeId="pl2"/>
 			<channel id="powerL3" typeId="pl3"/>
-			<channel id="phases" typeId="pha"/>
-			<channel id="sessionChargeEnergyLimit" typeId="scl"/>
-			<channel id="sessionChargedEnergy" typeId="scs"/>
-			<channel id="totalChargedEnergy" typeId="eto"/>
 			<channel id="allowCharging" typeId="alw"/>
-			<channel id="cableCurrent" typeId="cbl"/>
-			<channel id="temperature" typeId="tmp"/>
-			<channel id="firmware" typeId="fmw"/>
-			<channel id="accessConfiguration" typeId="ast"/>
+			<channel id="cableCurrent" typeId="cblCurr"/>
+			<channel id="firmware" typeId="fwv"/>
```

Comment:
Is this really needed as a channel? Usually firmware version is provided as a property.

Reply:
Sorry, the difference is not clear for me. Can you give me a hint or link?

Reply:
https://www.openhab.org/docs/developer/bindings/#properties

Reply:
Sorry, this channel also existed before, so it's okay to leave it for compatibility.